commercetools-models/src/test/java/io/sphere/sdk/carts/ExternalTaxRatesIntegrationTest.java

```diff
@@ -145,14 +145,15 @@ public void setLineItemTaxRate() {
                 final String taxRateName = "special tax";
                 final double taxRate = 0.20;
                 final ExternalTaxRateDraft externalTaxRate =
-                        ExternalTaxRateDraftBuilder.ofAmount(taxRate, taxRateName, DE).build();
+                        ExternalTaxRateDraftBuilder.ofAmount(taxRate, taxRateName, DE).includedInPrice(false).build();
```

Comment:
I would have added this as an additional test to check as this flag can have a default value

Reply:
Yes, ideally I would have spent more time on a separate test. But we try to spent just the minimal time on the JVM SDK and that's why I didn't create a separate test.